.github/workflows/prepare-release.yml

```
@@ -26,6 +26,34 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - name: Validate version format
+        run: |
+          # Define the regular expression pattern for the version format.
+          version_pattern='^[0-9]+\.[0-9]+\.[0-9]+(-rc[0-9]+)?$'
```

I think this should be something like the following:
```suggestion
          version_pattern='^[0-9]+\.[0-9]+\.[0-9]+(-rcv?[0-9\.]+)?$'
```

Since [v1.0.0-rcv0011](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.76.1) we had to use the `rcv` prefix because of how Go sorts pre-release versions

---

Okay, thanks will make the changes as suggested by you, anything else?